guide/14-deep-learning/use_mistral_llm_for_text_classification_and_entity_recognition.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p> </p><p>In this guide we demonstrated the steps to initialize and perform inference using the Mistral LLM as a backbone with the <code>TextClassifier</code> and <code>EntityRecognizer</code> models in <code>arcgis.learn</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>